test/e2e/benchmark.spec.ts

```diff
@@ -120,9 +123,21 @@ it('benchmark results', () => {
 		})
 		.on('cycle', function (event: any) {
 			log += `${event.target}\n`;
+			totals.push([event.target.hz, event.target.name]);
+			if (event.target.name.includes('Trie')) {
+				trieCount++;
+			}
 		})
 		.on('complete', () => {
 			console.log(log);
+			totals.sort(([a], [b]) => b - a);
+			let i = 0;
+			for (; i < trieCount; i++) {
+				expect(totals[i][1]).toInclude('Trie');
+			}
```

Comment:
Here I make sure that, on the sorted array by ops/sec (descreasing), have all the Trie tests on the top